feature-libs/organization/order-approval/core/services/order-approval.service.ts

```diff
@@ -40,11 +40,13 @@ export class OrderApprovalService {
   }
 
   loadOrderApprovals(params?: SearchConfig): void {
-    this.userIdService.invokeWithUserId((userId) =>
-      this.store.dispatch(
-        new OrderApprovalActions.LoadOrderApprovals({ userId, params })
-      )
-    );
+    this.userIdService
+      .takeUserId(true)
```

Comment:
Previously we allowed every customer Id. Is it the case here that we can't use these endpoints as an anonymous user?